docs/contributing-guide/forem.md

```diff
@@ -84,9 +84,9 @@ You can learn more about our internal RFC process and how we use forem.dev
 
 ### How to contribute
 
-1. [Fork the project](https://docs.forem.com/getting-started/forking/) and clone
+1. Fork the project and clone
    it to your local machine. Follow the
-   [installation guide](https://docs.forem.com/installation/)!
+   [installation guide](https://developers.forem.com/getting-started/installation/mac)!
```

Comment:
Since we're directly linking to the Mac installation guide, I'd prefer if we could change the text to say: `Follow the installation guide, [for example, the Mac guide](https://...etc)`

Ideally we could link to the installation guide tree as a page itself, but unfortunately Docusaurus doesn't support that easily.